test/synthetics/latest_release_cdn.js

```diff
@@ -0,0 +1,144 @@
+          
+          // Our release tags might or not contain a 'v' prefix (1.0.0 vs v1.0.0)
+          var latestRelease = JSON.parse(body).tag_name.replace('v','');
+          checkPackagesInCDN(repo, latestRelease);
+      }
+    );
+  });
+}
+
+function checkPackagesInCDN(repo, latestRelease){
+    // need to consider arm64 vs aarch64 vs noarch alternatives
+    var archs = [
+     {
+       archInUrl: 'x86_64',
+       archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+    }];
+    if (arm64Unsupported.indexOf(repo) < 0){
+      archs.push({ 
+        archInUrl: 'x86_64', 
+        archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+      });
+    } 
+
+    // except from infra-agent, all OHIs have consistent repos and package names.
+    var packagePrefix = repo === 'infrastructure-agent' ? 'newrelic-infra' : repo;
+
+    // TODO: add checks for all OS versions (blocked until all OHIs are released) 
+    // TODO: add DockerHub images checks
+    checkSLES(packagePrefix, latestRelease, archs);
+    checkAL2(packagePrefix, latestRelease, archs);
+    checkRHEL(packagePrefix, latestRelease, archs);
+    checkDeb(packagePrefix, latestRelease, archs);
+
+    if (windowsUnsupported.indexOf(packagePrefix) < 0){
+      checkWindows(packagePrefix, latestRelease);
+    }
+}
+
+function checkSLES(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.sles12.4.${archs[i].archInPackage}.rpm` : 
+         `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    var downloadServerURI = `${downloadServerBaseURI}/linux/zypp/sles/12.4/${archs[i].archInUrl}/${packageStructure}`; 
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkAL2(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.amazonlinux-2.${archs[i].archInPackage}.rpm` : 
+      `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    
+    var downloadServerURI = `${downloadServerBaseURI}/linux/yum/amazonlinux/2/${archs[i].archInUrl}/${packageStructure}`;
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkRHEL(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.el8.${archs[i].archInPackage}.rpm` :
+      `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    
+    var downloadServerURI = `${downloadServerBaseURI}/linux/yum/el/8/${archs[i].archInUrl}/${packageStructure}`; 
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkDeb(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var arch = archs[i].archInPackage === 'x86_64' ? 'amd64' : archs[i].archInPackage;
```

Comment:
for apt we also should check that package exists in metadata - http://download.newrelic.com/infrastructure_agent/linux/apt/dists/groovy/main/binary-amd64/Packages

Reply:
good point, I thought about this but left it out of scope, I'll add it in the next iteration.